android/storage.md

```diff
@@ -501,6 +471,37 @@ TransferObserver observer = transferUtility.upload(
 
 To download the metadata, use the S3 `getObjectMetadata` method. See the [API Reference](http://docs.aws.amazon.com/AWSAndroidSDK/latest/javadoc/com/amazonaws/services/s3/AmazonS3Client.html#getObjectMetadata%28com.amazonaws.services.s3.model.GetObjectMetadataRequest%29) and [Object Key and Metadata](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html) for more information.
 
+### Transfer Utility Options
+
+You can use the `TransferUtilityOptions` object to customize the operations of the TransferUtility.
+
+#### TransferThreadPoolSize
+This parameter allows you to specify the number of transfers that can run in parallel. By increasing the number of threads, you will be able to increase the number of parts of a multi-part upload that will be uploaded in parallel. By default, this is set to 2 * (N + 1), where N is the number of available processors on the mobile device. The minimum allowed value is 2.
+
+```java
+TransferUtilityOptions options = new TransferUtilityOptions();
+options.setTransferThreadPoolSize(8);
+
+TransferUtility transferUtility = TransferUtility.builder()
+    // Pass-in S3Client, Context, AWSConfiguration/DefaultBucket Name
+    .transferUtilityOptions(options)
+    .build();
+```
+
+#### TransferNetworkConnectionType
```

Comment:
Is the TransferServiceCheckTimeInterval no longer required?

Reply:
Yes. we have removed the feature of the cron job that checks for transfers and marked this API as deprecated. Hence, I am removing it from the docs.